<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Car</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Style</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Range</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Transmission</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Rating</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Price Range</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Vehicle Type</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="CarlineEdit"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="StylelineEdit"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="RangelineEdit"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="VehicleTypelineEdit"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="PriceRangelineEdit"/>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="RatinglineEdit"/>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="OkButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="CancelButton">
     <property name="text">
      <string>CANCEL</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="TransmissionlineEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
